src/main/resources/avro/common.avdl

```diff
@@ -2,6 +2,325 @@
 /**
 This protocol defines common types used in the other GA4GH protocols. It does
 not have any methods; it is merely a library of types.
+
```

Comment:
and can we *please* do consistent indentation and not special
case namespace to not be indented.  This makes it harder to read
and breaks automatic indention modes.  This kind of special-case
formatting to save columns when out of style a long time ago.

Frank Austin Nothaft <notifications@github.com> writes:
> In src/main/resources/avro/common.avdl:
> 
> > @@ -2,6 +2,325 @@
> >  /**
> >  This protocol defines common types used in the other GA4GH protocols. It does
> >  not have any methods; it is merely a library of types.
> > +
> 
> Small nit, but it looks like the indentation on this documentation is off.
> Should be a 2 space indent.
> 
> —
> Reply to this email directly or view it on GitHub.*
>